app/extensions/git_store/__init__.py

```diff
@@ -1144,14 +1133,7 @@ def update_asset_symlinks(
                 asset.update_symlink(local_asset_filepath)
                 asset.set_derived_meta()
 
-                log.debug(local_asset_filepath)
-                log.debug('\tAsset         : {}'.format(asset))
-                log.debug('\tSemantic GUID : {}'.format(asset.semantic_guid))
-                log.debug('\tMIME type     : {}'.format(asset.mime_type))
-                log.debug('\tSignature     : {}'.format(asset.magic_signature))
-                log.debug('\tSize bytes    : {}'.format(asset.size_bytes))
-                log.debug('\tFS xxHash64   : {}'.format(asset.filesystem_xxhash64))
-                log.debug('\tFS GUID       : {}'.format(asset.filesystem_guid))
+                log.debug(f'Created Asset {asset}')
```

Comment:
This one line contains all of the useful stuff from all of the above is a much more concise format.